styles.css

```diff
@@ -307,35 +340,57 @@ h4 {
 
 .modal-content {
   color:black;
-  background-color: #889CB9;
-  margin: 15% auto; /* Center it vertically and horizontally */
-  padding: 20px;
+  background-color: #FFF;
+  margin:0 auto;
+  padding: 16px;
   border: 1px solid #888;
   width: 80%; /* Adjust width as needed */
+  margin-top: 72px;
   max-width: 500px;
   text-align: center;
-  border-radius: 15px; /* This adds rounded corners */
+  border-radius: 16px;
   box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2); /* Optional: Add a shadow to make the modal look elevated */
+  position: relative;
+}
+
+.modal-content h2 {
+  color: #405779;
+  margin-top: 0;
 }
 
 .modal-button {
-  color: black;
+  color: #fff;
+  background-color: #405779;
   font-size: 14px;
   font-weight: bold;
-  border-radius: 5px; /* This adds rounded corners */
+  border-radius: 999px;
+  border: none;
+  min-height: 32px;
+  padding: 0 8px;
 }
 
 .close-button {
+  display: block;
+  width: 32px;
+  height: 32px;
+  display: grid;
+  align-items: center;
+  justify-items: center;
+  position: absolute;
+  top: 8px;
+  right: 8px;
+  border-radius: 999px;
+  border: none;
+  background-color: transparent;
   color: black;
-  float: right;
   font-size: 28px;
+  line-height: 1;
```

Comment:
But because we're using a `x` in this button and not an Icon.. we need to adjust for the height of the font